maven-scm-test/src/main/java/org/apache/maven/scm/ScmTestCase.java

```diff
@@ -369,7 +362,7 @@ public void assertCommandLine( String expectedCommand, File expectedWorkingDirec
 
     /**
      * @param cmd the executable to run, not null.
-     * @return <code>true</code>
+     * @return true iff the command is on the path
```

Comment:
Typo

Reply:
iff is math notation for "if and only if"

Reply:
Don't expect people to know that....

Reply:
Surprisingly, I have seen this today as IFF. Why not use this one to avoid typo resemblance.